quantum/keycodes.h

```diff
@@ -713,7 +713,41 @@ enum qk_keycode_defines {
     QK_AUTOCORRECT_ON = 0x7C74,
     QK_AUTOCORRECT_OFF = 0x7C75,
     QK_AUTOCORRECT_TOGGLE = 0x7C76,
+    QK_TRI_LAYER_1_3 = 0x7C77,
```

Comment:
This file should not be touched, anymore.  `qmk generate-keycodes` should be used to update this file.

Reply:
`qmk generate-keycodes` generates this file, it would be expected to be touched (not manually) when modifying keycodes.